packages/markitdown/src/markitdown/converters/_markdownify.py

```diff
@@ -17,6 +17,7 @@ class _CustomMarkdownify(markdownify.MarkdownConverter):
 
     def __init__(self, **options: Any):
         options["heading_style"] = options.get("heading_style", markdownify.ATX)
+        self.keep_data_uris = options.pop("keep_data_uris", False)
```

Comment:
likewise, we can

```suggestion
        options["keep_data_uris"] = options.get("keep_data_uris", False)
```
by this way, we have:
- Default value in case the option is not supplied in upstream objects
- Keep the implementation consistent

What do you think ?

Reply:
Absolutely! 🫡

Reply:
I have modified it in the form of kwargs passing parameters. For the sake of uniformity, I used the kwargs passing parameter form in all places where _CustomMarkdownify is initialized, and also reserved adjustment space for adding more optional parameters later.

Reply:
Since the command line parameter parser must declare parameters before parsing, kwargs cannot be used, so __main__.py still uses a fixed actual parameter passing method.